src/components/Pricing/PlanComparison/index.tsx

```diff
@@ -43,26 +37,23 @@ const sections = [
                 name: 'Plan benefits',
                 tiers: {
                     'PostHog Cloud': 'Scales as needed, Constant price',
-                    'Open source': 'Best for low volumes (~10k monthly users), Use with a small team',
-                    Free: 'Free, but limited to 1 million users',
-                    Scale: 'Cheaper at scale',
+                    'Open source': 'Great for small teams',
+                    Scale: 'Cheaper at scale, Enterprise features',
                 },
             },
             {
                 name: 'Pricing',
                 tiers: {
                     'PostHog Cloud': 'Free (up to 1 million events), then $0.000225/event',
                     'Open source': 'Free',
-                    Free: 'Free',
                     Scale: '$0.000225/event, $2k/mo minimum. (Discounts after 10 mil events)',
                 },
             },
             {
                 name: 'Scales to...',
                 tiers: {
                     'PostHog Cloud': 'Millions of users/mo',
-                    'Open source': '~10k users/mo (limited by database) ',
-                    Free: '1 million users/mo',
+                    'Open source': '~1m users/mo (limited by database) ',
```

Comment:
Not limited by database any more

Reply:
could add FAQ on "can I use PostHog open source with >1M users" to explain yes but we don't recommend it